File: app/src/main/java/com/joyhonest/sports_camera/MyNode.java
package com.joyhonest.sports_camera;

import android.graphics.Bitmap;
public class MyNode {
    public static final int Status_downloaded = 2;
    public static final int Status_downloading = 1;
    public static final int Status_waitdownload = 3;
    public static final int Stauts_normal = 0;
    public static int TYPE_Local = 0;
    public static int TYPE_SD = 1;
    public Bitmap bitmap;
    public int nGetType;
    public int nLength;
    public int nOp;
    public long nPre;
    public int nStatus;
    public int nType;
    public String sPath;
    public String sSDMp4name;
    public String sText;
    public String sUrl;

    public MyNode() {
        this.nOp = 0;
        this.nStatus = 0;
        this.nType = TYPE_Local;
        this.bitmap = null;
        this.sPath = "";
        this.sUrl = "";
        this.sText = "";
        this.sSDMp4name = "";
        this.nOp = 0;
        this.nLength = 0;
    }

    public MyNode(int i) {
        this.nOp = 0;
        this.nStatus = 0;
        this.nType = i;
        this.bitmap = null;
        this.sPath = "";
        this.sUrl = "";
        this.sText = "";
        this.sSDMp4name = "";
        this.nOp = 0;
        this.nLength = 0;
    }
}

File: app/src/main/java/com/joyhonest/sports_camera/MyNode.kt
package com.joyhonest.sports_camera

import android.graphics.Bitmap

class MyNode {
    var bitmap: Bitmap?
    var nGetType = 0
    var nLength: Int = 0
    var nOp: Int
    var nPre: Long = 0
    var nStatus: Int
    var nType: Int
    var sPath: String
    var sSDMp4name: String
    var sText: String
    var sUrl: String

    constructor() {
        nOp = 0
        nStatus = 0
        nType = TYPE_Local
        bitmap = null
        sPath = ""
        sUrl = ""
        sText = ""
        sSDMp4name = ""
        nOp = 0
        nLength = 0
    }

    constructor(i: Int) {
        nOp = 0
        nStatus = 0
        nType = i
        bitmap = null
        sPath = ""
        sUrl = ""
        sText = ""
        sSDMp4name = ""
        nOp = 0
        nLength = 0
    }

    companion object {
        const val Status_downloaded = 2
        const val Status_downloading = 1
        const val Status_waitdownload = 3
        const val Stauts_normal = 0
        var TYPE_Local = 0
        var TYPE_SD = 1
    }
}